### PR TITLE
For Issue 158 - getting relationships

### PR DESCRIPTION
**Pull request by aljachimiak**
This is the beginning of satisfying Issue #158.
In this PR you'll see this:
- Refactor out some fixtures for future testing
- Create some fixtures for future testing
- **Adds route and controller for `/collections/:id/relationships/:relationshipKey`**
- Tests the new route with all of the new test fixtures

### CODE REVIEW

File: lib/services/catalog/controllers/catalog-item-relationship-controller.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+		// 	args.include = req.query.include.split(',');
+		// }
+
+		// todo - this query may be improved to be relationship aware
+		this.bus
+			.query({role: 'catalog', cmd: 'fetchItem'}, args)
+			.then(resource => {
+				if (resource.relationships[relationshipKey]) {
+					res.body = resource.relationships[relationshipKey];
+					res.status(200);
```
Comment (aljachimiak) [inline lib/services/catalog/controllers/catalog-item-relationship-controller.js]: Setting the status here is undetectable in tests.

File: lib/services/catalog/index.js [hunk 2]
```diff
@@ -136,6 +138,16 @@ module.exports = function (bus, options) {
 				}}),
 				CatalogItemController.create({bus, type})
 			);
+
+			if (type === 'collection') {
```
Comment (aljachimiak) [inline lib/services/catalog/index.js]: At this point I've only added this for collections.  Wasn't sure what other types would get this route available to them.

**Comment by aljachimiak** [comment]
@kixxauth , @blainsmith , @brkattk  - I'd appreciate a few eyeballs on this.

Thanks!

File: lib/services/catalog/index.js (hunk 2 above)
Comment (brkattk) [inline lib/services/catalog/index.js]: All resources can contain relationships

**Comment by blainsmith** [comment]
Looks good, but like @brkattk said this should be allowed for any type.

**Pull request merged**